ledger/complete/wal/checkpoint_v6_reader.go

```diff
@@ -126,7 +129,11 @@ func filePaths(dir string, fileName string, subtrieLevel uint16) []string {
 
 // readCheckpointHeader takes a file path and returns subtrieChecksums and topTrieChecksum
 // any error returned are exceptions
-func readCheckpointHeader(filepath string, logger *zerolog.Logger) ([]uint32, uint32, error) {
+func readCheckpointHeader(filepath string, logger *zerolog.Logger) (
+	checksumsOfSubtries []uint32,
```

Comment:
I intended to have this verbose name, it is to ensure it's not being used and only for definition.